README.md

```diff
@@ -1,2 +1,180 @@
-# conditionalInstall
-Install npm packages based on various conditions
+# conditional-install
+
+A utility to install npm packages based on certain conditions.
+ 
+Why?
+--------------------
+
+The original motivation for this package is to install the proper version of
+the jest unit testing library.
+
+Some of the ilib packages supported node 10 through the
+current latest version and used jest as their unit testing library. The only
+problem is that the latest version of jest did not run on node 10 through 13
+because it was rewritten with ES6 syntax in jest@27.0.0. The tests, which were
+not particularly fancy and did not use any advanced features, ran fine
+with jest@26.6.2 on older versions of node but would not work with the latest
+jest because it wasn't transpiled to support older versions of node.
+
+What was needed was a way to install jest@26 for older versions of node and
+jest@29 for the newer ones. There was no easy, elegant way to do that.
+
+Hence this conditional install package.
+
+How Does it Work?
+--------------------
+
+Usage is simple. Basically, modify your package.json to include conditional
+dependencies, and then run this package as a postinstall script.
+
+Here is what the package.json would look like:
+
+```json
+{
+    "name": "mypackage",
+    "scripts": {
+        "postinstall": "node conditional-install"
+    },
+    "devDependencies": {
+        "conditional-install": "^1.0.0"
+    },
+    "conditionalDependencies": {
+        "process.version >= 14.0.0": {
+            "jest": "^29.0.0"
+        },
+        "process.version < 14.0.0": {
+            "jest": "^26.0.0"
+        }
+    }
+}
+```
+
+The postinstall script is run automatically by `npm` or `yarn` after the
+regular installation is complete. If you are using `npm-ci`, it does not, so
+you will have to run `npm-ci run postinstall` explicitly in order to
+install the conditional dependencies.
+
+The `conditionalDependencies` object contains properties that are
+expressions to test against the current version of node. Inside of the
+value object is a list of dependencies to install if the value of that
+expression is true. All dependencies inside of the value are installed.
+The versions are specified with the same syntax as regular, dev, or peer
+dependencies.
+
+When the dependencies are installed, they are installed using the current
+package manner, but in such a way that the new dependencies are not saved
+to the package.json.
+
+Conditional Expressions
+----------------------
+
+The expressions are built on the hard work of
+(node-compat-table)[https://github.com/williamkapke/node-compat-table].
```

Comment:
should [node-compat-table](https://github.com/williamkapke/node-compat-table).
to be displayed properly
square brackets brackets

Reply:
fixed